fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -70,6 +70,31 @@ get_jar_file() {
   fi
 }
 
+ceiling() {
+  awk -vnumber="$1" -vdiv="$2" '
+    function ceiling(x){
+      return x%1 ? int(x)+1 : x
+    }
+    BEGIN{
+      print ceiling(number/div)
+    }
+  '
+}
+
+# Based on the cgroup limits, figure out the max number of core we should utilize
+max_cores() {
```

Comment:
container options are just for JVM options no?  This is trying to set an ENV var.